npm/src/debug-service/debug-service.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  BreakpointSpan,
+  DebugService,
+  StackFrame,
+} from "../../lib/node/qsc_wasm.cjs";
```

Comment:
This debug-service.ts file is a platform agnostic file (i.e. is imported in code that runs on Node or Browser). It should only import types from the platform specific libs. By importing values (such as the classes here) it is sucking in the platform specific JavaScript imports into the build for either platform.

We should only import actual code from the platform-specfic libs in the platform specific source files (such as the -node, -browser files, or the platform specific entry points of browser.ts or main.ts)

Reply:
Interestingly, as you are only using these imports in type positions, you can change this import statement to be `import type {...`, and that way it will ensure no actual runtime code gets sucked in, and it still builds fine.